packages/common/src/services/cache/InMemoryCacheBackend.ts

```diff
@@ -0,0 +1,18 @@
+import { CacheBackend } from './Cache'
+import { NestedDict } from './NestedDict'
+
+export class InMemoryCacheBackend implements CacheBackend {
+  dict: NestedDict
+
+  constructor(data: NestedDict['data'] = {}) {
+    this.dict = new NestedDict(data)
+  }
+
+  read() {
```

Comment:
We spoke recently about explicitly stating the return type of the function. I'm not sure if it also applies to a short function like this one, just pointing this out.

Reply:
This is interface implementation, so I don't think that's necessary - types are already specified there.